deployment/ansible/roles/nyc-trees.app/templates/nginx-nyc-trees-app.conf.j2

```diff
@@ -6,6 +6,20 @@ server {
 
   access_log /var/log/nginx/nyc-trees-app.access.log logstash_json;
 
+  location /favicon.ico {
```

Comment:
I'm pretty sure it was working when I was making requests to /favicon.ico but I'll definitely double check. I'm having the same issue with github as you. The only explanation I can think of is that this is actually a `.png` file renamed to `favicon.ico`. Seems like that should work as per http://en.wikipedia.org/wiki/ICO_%28file_format%29.